### PR TITLE
Search & handlebars <head> fix

### PR DESCRIPTION
**Pull request by merezal**
Also has search

### CODE REVIEW

File: public/index.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+var search_box = document.querySelector('#nav_search_box');
+search_box.addEventListener('change', search);
+function search(event){
+    var url='/search/'+search_box.value.replace(/\s+/g, '').toLowerCase();
+    console.log('URL requested:', url);
+    window.location.replace(url);
```
Comment (julianfortune) [inline public/index.js]: I think we should use a different function than replace. According to Mozilla replace breaks hitting the back button  https://developer.mozilla.org/en-US/docs/Web/API/Location/replace . I think we should change this to assign https://www.w3schools.com/jsref/met_loc_assign.asp.

File: public/style.css [hunk 2]
```diff
@@ -71,6 +71,7 @@ input.nav_search_box{
   min-width:300px;
   min-height: 30px;
   border-radius: 10px;
+  padding:0 10px 0 10px;
```
Comment (julianfortune) [inline public/style.css:74]: Love this formatting, hope we can spread this everywhere over time!

**Review by julianfortune — CHANGES_REQUESTED** [review]
Excited to get this in! Just one change.

**Comment by julianfortune** [comment]
Also, is there a way to specify different js files for each page? I think we’re going to need different js on the view quiz page, edit/create page and other pages. Obviously that is *in addition* to the search javascript cause every page needs that.

**Comment by julianfortune** [comment]
Also, what do you think about naming 
`index.js` to `search.js` for clarity?

File: public/index.js (hunk 1 above)
Comment (merezal) [reply public/index.js]: Done

**Comment by merezal** [comment]
> Also, is there a way to specify different js files for each page? I think we’re going to need different js on the view quiz page, edit/create page and other pages. Obviously that is _in addition_ to the search javascript cause every page needs that.

I think what we could do is have:
{{each js}}
     <script src="{{javascript_file}}" charset="utf-8" defer></script>
{{/each}}

and then pass the needed js in server.js for each page

**Review by julianfortune — APPROVED** [review]
This looks excellent. Great work Michael 👏🏼👏🏼👏🏼. We'll eventually add more to the UI of the search results to match the featured quizzes boxes (and obvi we need to actually implement search on the backend), but all the code so far looks excellent. The templating changes are great, and being able to change js will make life super easy. 💯

**Pull request merged**